commands/create/cluster/command.go

```diff
@@ -154,6 +155,12 @@ Examples:
 )
 
 func init() {
+	initFlags()
```

Comment:
Can you explain this initialization and the resetting below please?

Reply:
Flags initialization is in `initFlags` so it can be triggered from tests.

`Command.ResetFlags()` is called in `initFlags` in order to make sure that flags are in the same state whenever the flags are initialized. Without this, tests issuing `SetFlags` or `ParseFlags` would set flags which would remain persisted, so in tests we would have to execute `ResetFlags` explicitly. I find this the safer way.